### PR TITLE
api: addition of batch encoding functions

### PR DESCRIPTION
**Pull request by switowski**
Requires #3

### CODE REVIEW

File: cds_sorenson/api.py [hunk 1]
```diff
@@ -43,78 +44,51 @@ def _str_(self):
...
-            "Failed to send encoding request to the server. Received "
-            "code: {0}. Make sure all parameters are set and permissions are "
-            "correct".format(response.status_code)
-        )
+        # something is wrong - sorenson server is not responding or the
+        # configuration is wrong and we can't contact sorenson server
+        raise SorensonError(response.status_code)
+
+
+def batch_start_encoding(input_files, presets_names):
```
Comment (egabancho) [inline cds_sorenson/api.py]: I don't see the need of sending several files to transcode at once, our use case goes more to given **one** master file generate **several** slaves using a given list of presets.

What about changing the API to something like:

``` python
def batch_start_encoding(input_file, *presets):
```

Comment (switowski) [reply cds_sorenson/api.py]: Good point, that could also help us avoid problems in the future (which job id belongs to which file).
@drjova can you confirm that sending one master file for encoding will be the only use case?

Comment (drjova) [reply cds_sorenson/api.py]: Yep, only one master file!

**Comment by switowski** [comment]
Done, @drjova can you take a look?

**Comment by drjova** [comment]
Could you please update the commit message :P 🦄 

WDYT about `api: addition of batch encoding functions`?

**Comment by switowski** [comment]
Right, sorry, forgot the commit message. Now it's fixed.

**Comment by drjova** [comment]
@switowski thanks LGTM!

**Pull request merged**